resources/functionMap.php

```diff
@@ -4729,7 +4729,7 @@
 'Imagick::getColorspace' => ['Imagick::COLORSPACE_*'],
 'Imagick::getCompression' => ['Imagick::COMPRESSION_*'],
 'Imagick::getCompressionQuality' => ['int'],
-'Imagick::getConfigureOptions' => ['string'],
+'Imagick::getConfigureOptions' => ['array'],
```

Comment:
Would be great to be more specific here, is it `array<uppercase-string, string>`?

Reply:
@ondrejmirtes I changed it here https://github.com/phpstan/phpstan-src/pull/3801/commits/e92d4786ed6654e39d3d199f8a2178c8d6a23780

It's not just uppercase strings.